fortanix-vme/fortanix-vme-runner/src/main.rs

```diff
@@ -1,7 +1,11 @@
 use fortanix_vme_runner::Server;
 use fortanix_vme_abi::SERVER_PORT;
+use std::io::ErrorKind;
 
 fn main() {
-    let (server_thread, _port) = Server::run(SERVER_PORT).expect("Server failed");
-    server_thread.join().expect("Server panicked");
+    match Server::run(SERVER_PORT) {
+        Ok((server_thread, _port))                   => server_thread.join().expect("Server panicked"),
+        Err(e) if e.kind() == ErrorKind::AddrInUse   => println!("Server failed. Have you already have a runner running on vsock port {}? Error: {:?}", SERVER_PORT, e),
```

Comment:
Nit: this message should probably be `"Server failed. Do you already have a runner running on vsock port {}? Error: {:?}"`.